react/AccordionItem/AccordionItem.test.js

```diff
@@ -53,7 +53,11 @@ describe('AccordionItem', () => {
       const button = renderedElement.querySelector('button');
 
       ReactTestUtils.Simulate.click(button);
-      expect(mockOnOpen).toHaveBeenCalledTimes(1);
+
+      // wait for useEffect
+      requestAnimationFrame(() => {
+        expect(mockOnOpen).toHaveBeenCalledTimes(1);
```

Comment:
hmm isnt this asynchronous , dont u have to await this?
same as below

Reply:
Good catch man. Added a `done()` call

Reply:
nice!